### ISSUE TITLE
when translating (to german) wrong "classes" are added

### ISSUE THREAD

**Issue by aoloe** [scratchblocks/scratchblocks#276]
if i translate

```
when flag clicked
forever
    move (30) steps
    play drum [1 v] for (0.5) beats
    move (-30) steps
    play drum [2 v] for (0.5) beats
```

i get

```
Wenn die grüne Flagge angeklickt
wiederhole fortlaufend 
  gehe (30) er-Schritt :: motion
  spiele Schlaginstrument [1 v] für (0.5) Schläge :: music
  gehe (-30) er-Schritt :: motion
  spiele Schlaginstrument [2 v] für (0.5) Schläge :: music :: control
end
```

- the `:: music` "class" is not needed,
- and the `:: control` one is wrong...
- `:: motion` is not needed either...

i got this effect also with other translations.

(otherwise, amazing work! : - )

**Comment by aoloe** [comment]
```
move (3) steps
```

also  becomes

```
gehe (3) er-Schritt :: motion :: control
```

**Comment by tjvr** [comment]
This should be fixed by #285 -- please open a new issue if there are still problems. :)
